app/models/pieces/king.rb

```diff
@@ -1,4 +1,23 @@
 class King < Piece
+  def can_castle?(rook)
+    return false unless first_move == 'true'
```

Comment:
if you change first_move to a `boolean`.

Then you could do. `return false unless first_move?`